src/command_modules/azure-cli-interactive/tests/test_feedback.py

```diff
@@ -53,10 +53,10 @@ def test_heuristic(self):
     def test_update_freq(self):
         """ tests updating the files for frequency """
         fh.update_frequency = self.norm_update
+        fh.FREQUENCY_PATH = os.path.join(shell_config(), 'mock_freq.json')
```

Comment:
But I want to cover the case where the file doesn't exist, in which case, I want to be able to control where the file path is. I don't think the making a temporary file will allow that.